pkg/kv/bulk/sst_batcher_test.go

```diff
@@ -41,8 +41,8 @@ func TestAddBatched(t *testing.T) {
 	t.Run("batch=default", func(t *testing.T) {
 		runTestImport(t, 32<<20)
 	})
-	t.Run("batch=1", func(t *testing.T) {
```

Comment:
This test is no longer valid since a 1byte max buf size means we can't grow the kvBuf to be more than that, which means the single KV doesn't fit in that buffer.